docs/zh-cn/components/helpers.md

```
@@ -1,41 +1,99 @@
+## 配置与自动加载
+
+此包通过 Composer 自动加载函数文件。其配置提供器不会注册或发布任何配置，因此无需额外设置。
+
+除 `call` 定义在 `FriendsOfHyperf\Helpers\Command` 命名空间外，其他函数均定义在
+`FriendsOfHyperf\Helpers` 命名空间。
+
+## 函数参考
+
+| 函数 | 签名与行为 |
+| --- | --- |
+| `app` | `app(null|string|callable $abstract = null, array $parameters = [])`：从容器解析服务；将可调用值转换为 `Closure`。 |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)